### ISSUE TITLE
Import from files with dublicate TIDs

### ISSUE THREAD

**Issue by mbigler74** [claeis/ili2db#459]
Force Import with errors like:
============================================================================================================
checking basket SIA405_LKMap.SIA405_LKMap bid=1 ...
============================================================================================================
*** ERROR *** CHECK(errid=20) duplicate transferid, TID=4163CCE0-94F8-49DE-8B12-BF24764E20FB
              LINE=476,TOPIC=SIA405_LKMap,CLASS=LKLinie,TID=4163CCE0-94F8-49DE-8B12-BF24764E20FB
*** ERROR *** CHECK(errid=70) UNIQUE constraint OBJ_ID (values=4163CCE094F849DE) violated, see also line 475
              LINE=476,TOPIC=SIA405_LKMap,CLASS=LKLinie,TID=4163CCE0-94F8-49DE-8B12-BF24764E20FB
*** ERROR *** CHECK(errid=20) duplicate transferid, TID=15E71B84-C0F9-4109-A2AC-CB5C6E357DFC
              LINE=1703,TOPIC=SIA405_LKMap,CLASS=LKObjekt_Text,TID=15E71B84-C0F9-4109-A2AC-CB5C6E357DFC

**Comment by edigonzales** [comment]
@mbigler74 Möchtest du, dass der Import trotz der Fehler funktioniert?

**Comment by claeis** [comment]
So ein --fixtid Modus wäre nett, damit man fehlerhafte Transferdateien via DB/GIS fixen kann

**Comment by mbigler74** [comment]
@edigonzales Ja, ich habe im Moment der Fall, dass bei einem Batchimport von LKMap Daten, lücken entstehen, da Daten mit diesen Fehlern nicht import werden. Schöner wäre natürlich nur "gute" Daten zu erhalten.
